src/main/java/ugh/LegacyUghImplementation.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
```

Comment:
Most files in this project are licensed under the LGPL (Lesser GNU General Public License) so this file needs even be licensed under this license.

Reply:
I don’t like the idea to design my own license header. Would it be okay to use the GDZ header as it is in all other files?

Reply:
This should be ok. If we would change it, then we change all of them to the new header.